crates/spirv-std/src/lib.rs

```diff
@@ -109,7 +109,6 @@ pub(crate) mod sealed;
 pub mod vector;
 
 pub use self::sampler::Sampler;
-pub use crate::macros::Image;
```

Comment:
This should still be exported.

Reply:
Makes sense, can do

Imo, we should choose one canonical place and stick with it, whether that's inside or outside of the image module of spirv_std

Reply:
Documentation of re-exports is not copied into re-exports of the re-export (i.e. no matter how we use this without duplicating the doc comment from `spirv_std::image::Image!`, we'll only get the docs from the macro in spirv_std_macros).

One alternative would be to make a macro `Image` in `spirv_std` which just calls `spirv_std::macros::Image`, which should work. But it would be nicer just not to have this re-re-export at the root.